src/components/structures/SpaceHierarchy.tsx

```diff
@@ -210,6 +210,25 @@ const Tile: React.FC<ITileProps> = ({
         topic = room.topic;
     }
 
+    let topicSection: ReactNode | undefined;
+    if (topic) {
+        topicSection = (
+            <Linkify
+                options={{
+                    attributes: {
+                        onClick(ev: MouseEvent) {
+                            // prevent clicks on links from bubbling up to the room tile
```

Comment:
The old code appears to check for A tags specifically and here we don't - is that a deliberate change?

Reply:
The old code bound a handler on the parent div, this binds the handler on all the linkified elements (anchor tags)